### PR TITLE
terraform basic structure

### PR DESCRIPTION
**Pull request by HarvindG**
Included in this pull request: 

- Created 5 ECR task definitions - just need the image arns to complete these
- Created 2 ECR services for the website and dashboard
- Created an S3 bucket
- Created 2 EventBridge schedulers
- Created a Step Function
- Created IAM roles and policies needed for eventbridge to run ECS tasks and trigger step functions

### CODE REVIEW

**Comment by HarvindG** [comment]
Need to adjust env variables within each task definition to only include necessary variables

File: terraform/variables.tf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+variable "DB_USER" {
+  description = "Value of the username for the postgres rds instance"
+  type        = string
+  default = "value"
+}
+
```
Comment (tadawson23) [inline terraform/variables.tf:6]: Really nicely laid out

File: terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+
+#   target {
+#     arn      = "arn:aws:ecs:eu-west-2:129033205317:cluster/c9-ecs-cluster" # arn of the ecs cluster to run on
+#     # role that allows scheduler to start the task (explained later)
+#     role_arn = aws_iam_role.iam_for_ecs.arn
+
+#     ecs_parameters {
+#       task_definition_arn = aws_ecs_task_definition.c9-sale-tracker-rds-cleanup-task-def.arn
+#       launch_type         = "FARGATE"
+
```
Comment (tadawson23) [inline terraform/main.tf:595]: Can this be removed?

Comment (tadawson23) [reply terraform/main.tf:595]: Maybe once the RDS is done.

**Review by tadawson23 — APPROVED** [review]

**Pull request merged**